addons/test_mail/models/test_mail_models.py

```diff
@@ -159,6 +161,13 @@ def _track_subtype(self, init_values):
             return self.env.ref('test_mail.st_mail_test_ticket_container_upd')
         return super(MailTestTicket, self)._track_subtype(init_values)
 
+    def _related_res_partner_default_create_values(self, normalized_email):
+        values = super()._related_res_partner_default_create_values(normalized_email)
+        email_from_normalized = email_normalize(self.email_from)
```

Comment:
I just added a mail.test.ticket.bl model I think (or was it not merged ?). As it inherits from mail.thread.blacklist (globally it is the mail ticket model + blacklist mixin), you could use email_normalize on it instead if you want :)

Reply:
It seems not merged yet (found mail.test.ticket.el that inherits from it ?)